features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -211,6 +211,25 @@ internal class SessionReplayFeature constructor(
     // endregion
 
     internal companion object {
+
+        /**
+         * Session Replay storage configuration with the following parameters:
+         * * max item size = 4 MB,
+         * * max items per batch = 500,
+         * * max batch size = 10 MB, SR intake batch limit is 10MB
+         * * old batch threshold = 18 hours.
+         */
+        internal val STORAGE_CONFIGURATION: FeatureStorageConfiguration =
```

Comment:
isn't it better to take `DEFAULT` and call `copy` here, only overriding what is needed for SR?

Reply:
Hmmm....good question, not sure which one is better. I think I will do that.